src/registry/create.ts

```diff
@@ -0,0 +1,59 @@
+import { type SignerConfig } from "../helpers/connection.js";
+import { type ContractTransaction } from "../helpers/ethers.js";
+import { validateTableName } from "../helpers/parser.js";
+import { getContractAndOverrides } from "./contract.js";
+
+// Match _anything_ between create table and schema portion of create statement
```

Comment:
Ok, this is kinda crazy... but this is how I make it possible to extract a plain table name from something like:

```sql
CREATE TABLE my_table (...);
```
and combine it with the chainId information to create a new table name for a create table statement like this: `my_table_31337`. I ended up on a two part regex pattern, where we first take _anything_ that might be a name, and then parse it out as best as possible, dealing with escaping and things like that. It'll fail if you try to do anything particularly fancy with your table name. But things like [my_table], or `my_table` etc that many ORMs use should be safe.